### ISSUE TITLE
univention-moodle-group installation error and incorrect Teacher attribute scripting issue

### ISSUE THREAD

**Issue by rbrannath** [univention/cool-solutions#7]
While following the instructions at [https://help.univention.com/t/cool-solution-install-moodle-advanced-ldap-enrollment/21318], I encountered an error while executing the join script:

```
Calling joinscript 99MoodleGroupSetter.inst ...
2023-12-05 09:50:53.367472305+01:00 (in joinscript_init)
Joinscript 99MoodleGroupSetter.inst finished with exitcode 1
dpkg: Error processing package univention-moodle-group (--configure):
 'installed univention-moodle-group script of package post-installation' subprocess returned error code 1
Errors were encountered while processing:
 univention-moodle-group
```

The system diagnosis also indicated an error, stating that the package univention-moodle-group is corrupt.

By renaming the join script in /usr/lib/univention-install from "99MoodleGroupSetter.inst" to "99univention-moodle-group.inst," I was able to execute the join script without any issues.

Additionally, there was a minor yet critical copy-paste error. In the script, the attribute name for the Teacher is mistakenly labeled as "Student," resulting in the teacher attribute not being created.

Last but not least, there is a typo in the readme:

`uninvention-install univention-moodle-group` should be corrected to `univention-install univention-moodle-group`.

**Comment by admirableadmin** [comment]
Hey @rbrannath
Thank you for the contribution and the pull request #8 - I will look onto it.

First, renaming the join-script doesn't solve a problem:
* it has to be the same as in [debian/install#L1](https://github.com/univention/cool-solutions/blob/ucs-5.0/master/univention-moodle-group/debian/install#L1) and [debian/postinst#L20](https://github.com/univention/cool-solutions/blob/ucs-5.0/master/univention-moodle-group/debian/postinst#L20)
* By renaming, I guess, it was just a re-execution of the join script on your side. Maybe in your case a `univention-run-join-scripts` would have solved that too.  
* Could you please undo the rename of the inst-file in your pull request (and maybe send me the relevant `/var/log/univention/join.log` file on peichert@univention.de to search for the real problem)? 

Second, the wrong name of the extended-attribute and the typo in the readme are good catches. Thank you.

**Comment by rbrannath** [comment]
Already tried to run `univention-run-join-scripts` (without --force) and it stated the join script was already executed.
By adding the --force flag it was executed but not doing anything. It may was a problem related to my very system. I will send you the logs.

**Comment by admirableadmin** [comment]
Thanks for undo the rename. I cherry-picked your commits in our internal git, and we will release an update in the following days.

The issue was that the extended-attribute Teacher (univentionFreeAttribute19) was never created because of a duplicate "--set name=Student", this situation cant fixed by uninstalling the package. And even worse it result in package state "iF" on reinstalling, because the join-script was already executed (Joinscript 99MoodleGroupSetter.inst finished with exitcode 1). This have been fixed.

**Comment by admirableadmin** [comment]
We just released univention-moodle-group 1.0.0-5 with the patches, so I close this issue.
Thank you again for pointing this out
